Content.Shared/Zombies/SharedZombieSystem.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Content.Shared.Movement.Systems;
+
+namespace Content.Shared.Zombies;
+
+public abstract class SharedZombieSystem : EntitySystem
```

Comment:
you also need to add a non-abstract empty client-side system that inherits from this in order for it to work.